relay.go

```diff
@@ -169,8 +169,9 @@ const (
 
 // A Relayer forwards frames.
 type Relayer struct {
-	relayHost  RelayHost
-	maxTimeout time.Duration
+	relayHost      RelayHost
+	maxTimeout     time.Duration
+	maxConnTimeout time.Duration
```

Comment:
Should this be exposed for introspection as well?

Reply:
Good point, added introspection.